src/patterns/question-pages/default.njk

```diff
@@ -15,8 +15,14 @@ layout: layout-example.njk
   <div class="govuk-o-main-wrapper">
     <div class="govuk-o-grid">
       <div class="govuk-o-grid__item govuk-o-grid__item--two-thirds">
-
-        <form class="form" action="/url/of/next/page" method="post">
+      <!--
```

Comment:
Open to suggestions to make this less verbose whilst keeping it useful. Or if people think we can just omit it.

Reply:
The Prototype Kit is using the same mock address in the example pages, so at least we're being consistent.

Reply:
Yeah, I guess I'm wondering how important it is to provide an example of how the `<form>` tag might look in a prototype, and if it's worth such a long comment to explain it!

Reply:
@joelanman is setting `<form>` attributes correctly something that users struggle with when prototyping? I've added a comment here to try and help users use the correct form attributes, but I don't know if a) they'll actually see it and b) if it's overkill?

Reply:
hm tricky. Purely from a prototyping point of view I'd prefer to keep form actions being POST. We switched over a whole ago, the main reason being security - data submitted using GET can hang around all over the place: server logs, browser history, etc. It's also generally more correct from a REST point of view, though correctness is a secondary concern in prototyping :)

In terms of users struggling to set form attributes, I would say definitely yes, and I would very much doubt they would see the comments.

I think if we want to avoid broken submits, the best thing is to make the submit work - is that tricky in metalsmith?

Reply:
Leaving aside the prototype kit, my understanding is that it's generally more 'correct' for forms to POST onwards - should our documentation thus follow the convention and POST as well?

Summary - I'm with Joe - make the POST work. Could we do an autoredirect thing like we do on the kit?

Reply:
I think we're missing the point here. This PR is an update so when someone actually presses the submit button on the Design System it won't be sent `/url/of/next/page` which results in a 404.

@36degrees proposal was to explain how to use the form in a comment
```
        This example omits the 'action' and uses the 'get' method so that
        submitting the form just reloads the example. In most cases, your form
        tag will probably need to look something like this:
        <form action="/url/of/next/page" method="post">
```
and use GET so the example won't redirect the user when playing with the example in the Design System.

We can potentially use the POST directly and prevent the submit in the Design System examples using JavaScript, but that means we won't cover visitors with JavaScript disabled.

Reply:
I'd suggest each example POST to itself rather than that generic url. The example can then redirect and get the original page (or a mini success page with results and link back to example?)

Reply:
@alex-ju no I understand :) it's just I don't have confidence people prototyping will read that comment and understand it. We normally tell people to simply copy and paste examples into their prototypes.

Reply:
Exactly – because the Design System is built as a static site, the server doesn't know how to handle a `POST` here and so it falls back to an error page:

<img width="1392" alt="screen shot 2018-03-02 at 10 57 32gmt" src="https://user-images.githubusercontent.com/121939/36895921-92971434-1e08-11e8-8e7f-d39796752998.png">

Reply:
@36degrees can we add redirects at a hosting level somehow? Tell PaaS to redirect these POSTs to GETs?

Reply:
@36degrees can that be addressed somehow? It feels really odd to have examples using `GET`s.

Idea: could the examples be served from an app whilst the design system isn't? This might be useful in other ways too.